app/liquid_tags/youtube_tag.rb

```diff
@@ -1,5 +1,8 @@
 class YoutubeTag < LiquidTagBase
   PARTIAL = "liquids/youtube".freeze
+  # rubocop:disable Layout/LineLength
+  YOUTUBE_URL_REGEX = %r{https?://(www\.)?(youtube|youtu)\.(com|be)/(embed|watch)?(\?v=)?(/)?[a-zA-Z0-9_-]{11}((\?t=)?(\d{1,})?)?}
```

Comment:
Since this regex matches the host and domain part separately it will also match the incorrect `youtu.com` (youtube.be actually correctly redirects).